security/pkg/server/ca/server.go

```diff
@@ -101,6 +101,21 @@ func (s *Server) CreateCertificate(ctx context.Context, request *pb.IstioCertifi
 	return response, nil
 }
 
+// rootCertExpirationSeconds returns the unix timestamp when the root becomes expires.
+func rootCertExpirationSeconds(ca ca.CertificateAuthority) float64 {
+	rb := ca.GetCAKeyCertBundle().GetRootCertPem()
+	cert, err := util.ParsePemEncodedCertificate(rb)
+	if err != nil {
+		log.Errorf("Failed to parse the root cert: %v", err)
+		return -1
```

Comment:
is this a standard way of expressing error value? i am afraid when users really see this, they may feel their cert has already been expired (since it is -1 seconds) and get worried.

Reply:
That will be a internal error, cert corruption or our library to parse the cert, `util` pkg. False positve seems better than false negative. aka. it's better to be more cautious as opposed to miss potential real problem?

I add a description to this metrics name `We set it to negative in case of internal error.",`